app/src/main/res/values/strings.xml

```diff
@@ -639,6 +639,9 @@
     <string name="warning_switch_server">Switching Server: This will clear all mUzima data on your
         device, including any incomplete forms. Are you sure you want to change this setting?
     </string>
+    <string name="warning_session_timeout">Session Timeout: Could not set session timeout. The
```

Comment:
Hi @rezita Can you update translations for this string entry in the respective string.xml files

Reply:
Hello, I have added the translations. I am not sure if they are 100% correct. (I used google translate. My Swahili is not the best. :) )